### DISCUSSION TITLE
Dev server watch without reloading browser

### DISCUSSION THREAD

**Discussion by nielsbauman** [modernweb-dev/web#2401] [Q&A] [answered]
Hi there,

I know I can run `web-dev-server` with the `--watch` option to get it to recompile on file changes, but I want to avoid the automatic reload of my browser tabs.
I sometimes edit some CSS or run some JS in an open tab and then make a small change in my file which causes my temporary changes to be flushed due to the page reload.
Is there anyway I can enable the recompilation on file change but disable the browser reload?

Thanks

**Comment by thepassle** [comment, marked as answer]
Wds doesnt do any compilation. If you are using plugins that do some code transforms, those likely happen as the files are requested. It sounds like you just shouldnt use watch mode in this case.

**Reply by nielsbauman** [reply]
Ah my mistake, I think I got confused, you're right. Thanks for the quick answer!
